functions/indexes/indexes.py

```diff
@@ -0,0 +1,25 @@
+#
+# Copyright (c) 2023, RTE (http://www.rte-france.com)
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+#
+
+import requests
+import constant
+
+#
+# @author Hugo Marcellin <hugo.marcelin at rte-france.com>
+# @author Sylvain Bouzols <sylvain.bouzols at rte-france.com>
+#
+
+def get_nb_indexed_equipments():
+    return requests.get(constant.GET_STUDIES_INDEXED_EQUIPMENTS_COUNT, params={}).text
```

Comment:
I reckon params is optional, you can remove it if there's none of it

Reply:
done